verification/verify_gig_visuals.py

```
@@ -0,0 +1,97 @@
+        browser = p.chromium.launch(headless=True)
+        page = browser.new_page()
+
+        # 1. Load Game
+        print("Loading game...")
+        page.goto("http://localhost:5174")
+
+        # 2. Start New Game
+        print("Starting new game...")
+        try:
+            page.get_by_text("Start Tour", exact=True).click()
+        except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)